test/e2e/restart.go

```diff
@@ -68,6 +68,9 @@ var _ = Describe("Restart [Disruptive]", func() {
 
 	It("should restart all nodes and ensure all nodes and pods recover", func() {
 		nn := testContext.CloudConfig.NumNodes
+		// Sadly, this count must be kept up to date.  This total is 13 system pods
```

Comment:
at the very least this should probably be a constant, rather than tucked away inside a test case.